### PR TITLE
ZUI 75 trustpilot title

### PR DESCRIPTION
**Pull request by stephanie-senoner**
Changest trustpilot svg title to be dynamically set according to the rating:

https://jira.zopa/browse/ZUI-75

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Execute the development environment:
 pnpm dev
 ```
 
+Test
```
Comment (admmasters) [inline README.md:42]: Test succeeded?

Comment (poteirard) [inline README.md:42]: Typo?

Comment (stephanie-senoner) [reply README.md:42]: :( sorry - that was because I wasn't sure if it works when I changed the github user on my terminal

**Review by poteirard — APPROVED** [review]

**Review by khumbon — APPROVED** [review]

**Pull request merged**